### PR TITLE
Sprint 10 Release

### PR DESCRIPTION
**Pull request by lpendem**
This release PR includes the following stories completed in Sprint 10:

* https://github.com/nciinc/fs-permit-platform/issues/250
* https://github.com/nciinc/fs-permit-platform/issues/425
* https://github.com/nciinc/fs-permit-platform/issues/427
* https://github.com/nciinc/fs-permit-platform/issues/428
* https://github.com/nciinc/fs-permit-platform/issues/429
* https://github.com/nciinc/fs-permit-platform/issues/436
* https://github.com/nciinc/fs-permit-platform/issues/440
* https://github.com/nciinc/fs-permit-platform/issues/469
* https://github.com/nciinc/fs-permit-platform/issues/471
* https://github.com/nciinc/fs-permit-platform/issues/473
* https://github.com/nciinc/fs-permit-platform/issues/474
* https://github.com/nciinc/fs-permit-platform/issues/475
* https://github.com/nciinc/fs-permit-platform/issues/476
* https://github.com/nciinc/fs-permit-platform/issues/477
* https://github.com/nciinc/fs-permit-platform/issues/478
* https://github.com/nciinc/fs-permit-platform/issues/479
* https://github.com/nciinc/fs-permit-platform/issues/480
* https://github.com/nciinc/fs-permit-platform/issues/483
* https://github.com/nciinc/fs-permit-platform/issues/485
* https://github.com/nciinc/fs-permit-platform/issues/487
* https://github.com/nciinc/fs-permit-platform/issues/497
* https://github.com/nciinc/fs-permit-platform/issues/498
* https://github.com/nciinc/fs-permit-platform/issues/499

### CODE REVIEW

File: server/src/controllers/temp-outfitter.es6 [hunk 1]
```diff
@@ -16,16 +15,13 @@ const email = require('../email/email-util.es6');
...
 const tempOutfitter = {};
 
-/** Initialize our S3 bucket connection for file attachments */
-const s3 = new AWS.S3({
-  accessKeyId: vcapConstants.accessKeyId,
-  secretAccessKey: vcapConstants.secretAccessKey,
-  region: vcapConstants.region
-});
+
+const s3 = util.getS3();
```
Comment (lauraGgit) [inline server/src/controllers/temp-outfitter.es6]: is this now commonController? or is there still an utility controller?

File: server/src/controllers/temp-outfitter.es6 [hunk 2]
```diff
@@ -37,21 +33,17 @@ const translateFromClientToDatabase = (input, output) => {
...
-    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.extension
-      ? input.applicantInfo.eveningPhone.extension
-      : null;
+    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.extension ? input.applicantInfo.eveningPhone
+      .extension : null;
   output.applicantInfoEveningPhoneNumber =
-    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.number
-      ? input.applicantInfo.eveningPhone.number
-      : null;
+    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.number ? input.applicantInfo.eveningPhone.number :
```
Comment (lauraGgit) [inline server/src/controllers/temp-outfitter.es6]: this seems like it exceeds the linting line length?

File: server/src/controllers/temp-outfitter.es6 [hunk 3]
```diff
@@ -37,21 +33,17 @@ const translateFromClientToDatabase = (input, output) => {
...
+      null;
   output.applicantInfoEveningPhonePrefix =
-    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.prefix
-      ? input.applicantInfo.eveningPhone.prefix
-      : null;
+    input.applicantInfo.eveningPhone && input.applicantInfo.eveningPhone.prefix ? input.applicantInfo.eveningPhone.prefix :
```
Comment (lauraGgit) [inline server/src/controllers/temp-outfitter.es6]: same as above?

File: server/src/email/email-util.es6 [hunk 4]
```diff
@@ -63,9 +64,10 @@ emailUtil.sendEmail = (templateName, data, attachments = []) => {
     let html;
     let templateAttachments;
     if (template.html) {
-      html = `${htmlTemplate.forestService}<div class="body">${
-        template.html
-      }<p>Please do not reply to this message. This email message was sent from a notification-only address that cannot accept incoming email.</p></div>`;
+      html =
+        `${htmlTemplate.forestService}<div class="body">${
+          template.html
+        }<p>Please do not reply to this message. This email message was sent from a notification-only address that cannot accept incoming email.</p></div>`;
```
Comment (lauraGgit) [inline server/src/email/email-util.es6]: can you extend this via multi-line - the line length is long.

File: server/src/services/util.es6 [hunk 5]
```diff
@@ -250,22 +233,45 @@ util.getRandomString = length => {
...
+    return res.status(404).send();
+  } else {
+    return res.status(500).send();
+  }
+};
+
 util.request = request;
 
+util.getS3 = () => {
+  /** Initialize our S3 bucket connection for file attachments */
```
Comment (lauraGgit) [inline server/src/services/util.es6]: jsDoc statements are generally before the method

File: .circleci/config.yml [hunk 6]
```diff
@@ -54,8 +50,19 @@ jobs:
   run-e2e:
     machine: true
     parallelism: 4
+    environment:
+      - DATABASE_URL: "postgres://ubuntu@localhost/circle_test"
     steps:
       - checkout
+      - run:
+          name: install aws cli
+          command: sudo apt-get update && sudo apt-get install -y awscli
```
[the remark this answers is not in the document]
Comment (lpendem) [reply .circleci/config.yml]: yes, we can move it right before the `clean s3 bucket` script at the end @lauraGgit

File: .circleci/config.yml [hunk 7]
```diff
@@ -69,13 +76,30 @@ jobs:
...
+      - run:
+          name: build pa11y compose images
+          command: cd docker; ../.circleci/build-pa11y-image.sh
+      - run:
+          name: pa11y test
+          command: cd docker; ../.circleci/pa11y.sh
+
   staging-deploy:
     docker:
       - image: circleci/node:8.9.4
```
[the remark this answers is not in the document]
Comment (lpendem) [reply .circleci/config.yml]: @lauraGgit It appears that it has to be set separately for each docker based job.

File: .circleci/config.yml [hunk 8]
```diff
@@ -2,48 +2,44 @@ version: 2
 jobs:
   run-all-other-tests:
     machine: true
+    environment:
```
[the remark this answers is not in the document]
Comment (lpendem) [reply .circleci/config.yml]: @lauraGgit As of sprint-11, PLATFORM is being set in the commands, but it is being set that way until sprint-10

File: server/src/controllers/temp-outfitter.es6 (hunk 1 above)
Comment (lpendem) [reply server/src/controllers/temp-outfitter.es6]: @lauraGgit No. util is there as service, there's still common controller.

File: server/src/controllers/temp-outfitter.es6 (hunk 2 above)
Comment (lpendem) [reply server/src/controllers/temp-outfitter.es6]: @lauraGgit there's no linting limit on the server code, so I am configuring it to be 125. Frontend has  200 characters limit.

Comment (lauraGgit) [reply server/src/controllers/temp-outfitter.es6]: linting should conform to 18f's style guide which is based off of `airbnb`'s https://github.com/airbnb/javascript> they say 100 actually, but given how late we are in 125 should be ok https://github.com/airbnb/javascript#whitespace--max-len

**Pull request merged**